selfdrive/manager/manager.py

```diff
@@ -193,6 +215,7 @@ def main():
   if Params().get_bool("DoUninstall"):
     cloudlog.warning("uninstalling")
     HARDWARE.uninstall()
+  return None
```

Comment:
why was this added?

Reply:
mypy doesn't like implied returns, so it will error with either "Missing return statement" or "Return value expected" when adding any return type hints.